README.md

```diff
@@ -41,3 +41,5 @@ n = 10**5
 sleep_durations = [random.uniform(0, 5.0) for _ in range(n)]
 asyncio.run(run_multiple_sleeps(sleep_durations=sleep_durations))
 ```
+
+A complete demo of `tqdm_publisher` can be found in the `demo` directory, which shows how to forward progress updates from the same `TQDMPublisher` instance to multiple clients.
```

Comment:
Could you expand this details of how someone would run the demo? Do you they just call Python on the `demo/main.py` file and open the HTML file in any web viewer?

Reply:
Yeah you got it. I've updated the docs, but how would you want to specify in the demo's `requirements.txt` that it depends on an editable install of the repo's module?

Reply:
Why does it? So that they have a Python file to call?

One of the next things I was going to request (and this ought to be easy with the `pyproject.toml` build strategy) is to expose a CLI entrypoint to run the demo (which ought to then work no matter what the installation pathway)

Reply:
Just so it can use `tqdm_publisher`. Maybe there's a better way to reference the module locally?

Do you have any docs for the CLI entrypoint using `pyproject.toml`? Otherwise I'll dig and see if I can find anything that references that.

Reply:
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#creating-executable-scripts

> Just so it can use tqdm_publisher. Maybe there's a better way to reference the module locally?

There are, but you wouldn't want them in a demo script. If they've done `pip install tqdm_publisher`, it should work fine (to run the demo, they do need something installed somehow)

Reply:
Got it. Just added `tqdm_publisher` to the requirements and switched the base branch to add the executable script.